docs/reference/docs/block-until-refresh.asciidoc

```diff
@@ -0,0 +1,42 @@
+[[block-until-refresh]]
+== Block Until Refresh
```

Comment:
wondering if we should use the same `refresh=wait_until` , `refresh=immediate` options here. Thoughts?